llvm/test/CodeGen/Mips/nanomips/jumptable.ll

```diff
@@ -8,11 +8,11 @@ define i32 @test_jumptable(i32 %in) {
     i32 4, label %lbl4
   ]
 
-; CHECK: li $a1
-; CHECK: Li_NM
-; CHECK: lwxs $a0, $a0($a1)
-; CHECK: LWXS_NM
-; CHECK: jrc $a0
+; CHECK: la $a1
+; CHECK: LA_NM
+; CHECK: lbux $a0, $a0($a1)
```

Comment:
Guess I was not explicit enough. This test should continue to check the LWXS expansion of jump tables, by being compiled with `-mattr=disable-jump-table-opt`. Changing li -> la is correct, but changing lwxs -> lbux is contrary to the intention of this test.

Reply:
Okay, I understand now, I'll fix it. Thank you.